src/rustup-cli/rustup_mode.rs

```diff
@@ -463,6 +471,15 @@ pub fn cli() -> App<'static, 'static> {
                 SubCommand::with_name("default-host")
                     .about("The triple used to identify toolchains when not specified")
                     .arg(Arg::with_name("host_triple").required(true)),
+            )
+            .subcommand(
+                SubCommand::with_name("profile")
+                    .about("The default components installed")
+                    .arg(
+                        Arg::with_name("profile-name")
+                            .required(true)
+                            .help("minimal|default|complete"),
```

Comment:
Is this meant to be an exhaustive list of valid options?  If so, we should really have a `clap` validation function no?

Reply:
In fact, should it be via `Profile::names()` somehow?